app/poros/contributors.rb

```diff
@@ -0,0 +1,9 @@
+class Contributors
```

Comment:
why is this in the subfolder called `poros`

Reply:
PORO stands for Plain Old Ruby Object, and its where these things live in a rails app